sdk/scheduler/src/main/java/com/mesosphere/sdk/api/ConfigResource.java

```diff
@@ -26,14 +26,15 @@
  *
  * @param <T> The configuration type which is being stored by the framework.
  */
+@Singleton
```

Comment:
Why's this needed?

Reply:
(More out of curiosity than anything else. I haven't ever needed this annotation on resources.)

Reply:
solely to fix `WARNING: A resource, class com.mesosphere.sdk.api.ConfigResource, implements provider interface [interface javax.ws.rs.ext.ContextResolver] but does not explicitly define the scope (@Singleton, @PerLookup). The resource class will be managed as singleton.`